forms-web-app/src/views/appellant-submission/site-ownership-certb.njk

```diff
@@ -0,0 +1,66 @@
+{% extends "layouts/main.njk" %}
+
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+
+{% set ownsWholeSite = appeal.appealSiteSection.siteOwnership.ownsWholeSite %}
+
+{% block pageTitle %}
+  Site ownership | Appeal a householder planning decision
+{% endblock %}
+
+{% block content %}
+
+  <div class="govuk-grid-row">
+
+    <div class="govuk-grid-column-two-thirds">
+
+      {% if errors %}
+        {{ govukErrorSummary({
+          titleText: "There is a problem",
+          errorList: errorSummary
+        }) }}
+      {% endif %}
+
+      <form action="" method="post" novalidate>
+        <span class="govuk-caption-l"><span class="govuk-visually-hidden">Section. </span>Visiting the appeal site</span>
+        <div class="govuk-form-group">
+          <fieldset class="govuk-fieldset">
+            <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+              Have you told the other owners that you’re appealing?
+            </legend>
+            <div class="govuk-warning-text">
```

Comment:
this template does not use the nunjucks macros because of the requirement for the warning text added here, which is not possible with the macro.